### ISSUE TITLE
Apple TV keeps throwing error about device requirements

### ISSUE THREAD

**Issue by rogeriopvl** [Stremio/stremio-bugs#100]
So I open the latest version of Stremio on my macbook, and suddenly the Apple TV turns on and shows an error dialog "Device Requirements", where it says that "this airplay connection needs iOS 7.1...".

This dialog keeps popping up as long as I have Stremio running.

This happens without any special configuration, or action. To replicate this you just need to have an apple tv on the same wifi as your macbook and open streamio.

It seems to me that stremio is attempting to connect to the apple tv in background, but the airplay device verification "blocks" it.

**Comment by Ivshti** [comment]
Apple released an update to the Apple TV that disallows any third-party desktop applications from streaming to it. Unfortunately, we don't know how to (or whether it's possible)  to work around that yet

**Comment by rogeriopvl** [comment]
@Ivshti thanks for the reply.

So I would guess that stremio is doing some kind of "airplay scanning" in the background. Maybe that behaviour could be turned off (since it doesn't currently work) or toggled with a setting? Apple TV users can still use airplay mirroring to stream to the TV.

**Comment by easyme123** [comment]
I was facing the exact same issue. When I opened Stremio on mac mac my AppleTV would show an error claiming "device requirements" and etc. Then I realized that if i don't accept the "Stremio.app" and "StremioHelper.app" conections when I open the Streamio app , it would conect anyways and the error message don't pop-up.

(In my mac when I open Stremio i always get 2 prompts asking to accept connections for the app. I don't know if everyone gets this but I have the "accept connections automatically for transfered apps" option turned OFF on my mac firewall)
